### DISCUSSION TITLE
Export Option for Sectors & Systems

### DISCUSSION THREAD

**Discussion by HuntingPhoenix** [bcholmes/StarTrek2d20#65] [Ideas]
Your new sector generation function is great! I'd made a python script for this myself, but this is better!

I'd love to see an export option to be able to capture the output easily for notes. I'd suggest an image of the sector representation and a way to get some formatted text to copy/paste.

**Comment by bcholmes** [comment]
Yeah, it's a work in progress. It'll probably end up being a PDF export like other stuff on the site.

**Comment by HuntingPhoenix** [comment]
I think a pdf might be overkill, actually.  I think making the text and image be copy/paste might actually be enough.  Not sure if that's easy or hard to do though.
